files/common/lib/sysctl.d/50-override.conf

```diff
@@ -105,3 +105,26 @@ fs.inotify.max_user_watches = 32768
 # entirely here, to avoid the problem.
 #
 fs.leases-enable = 0
+
+#
+# Ubuntu 20.04 OS hardening changes for CIS benchmarking
+# ICMP Redirects, send routing information to other hosts is not
+# required because this host is not acting as a router.
+#
+# A system with multiple interfaces need net.ipv4.ip_forward
+# to forward the packets, which is not required.
+#
+net.ipv4.conf.all.send_redirects = 0
```

Comment:
[nit] - Should all net.ipv4 setting be placed at one place?

Reply:
Agree, They. could be divided into 4 sections, Thanks @ShibasishDelphix for quick review